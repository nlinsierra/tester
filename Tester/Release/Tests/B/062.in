3C 5C JC 7H QH 8D 3S QS TS KC JD KH 7C 2H 6C 5D 4S QD TC 3D TD KS 7S 4H KD
