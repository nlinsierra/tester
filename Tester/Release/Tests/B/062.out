27
2C 4C 8C 9C QC AC 2D 4D 6D 7D 9D AD 3H 5H 6H 8H 9H TH JH AH 2S 5S 6S 8S 9S JS AS
